packages/core/src/lib/actions/callback/oauth/callback.ts

```diff
@@ -129,6 +135,10 @@ export async function handleOAuth(
   let profile: Profile = {}
   let tokens: TokenSet & Pick<Account, "expires_at">
 
+  if (provider.authorization?.serverConform) {
+    as = await provider.authorization?.serverConform(as, codeGrantResponse)
+  }
+
```

Comment:
I'm having some trouble here. It might be the way my development environment is set up, but it seems to think `serverConform` isn't a part of the object.